### ISSUE TITLE
Swin-Transformer的收敛性验证

### ISSUE THREAD

**Issue by rentainhe** [Oneflow-Inc/models#270]
Swin-Transformer的验证
- 对于input的数据增强先采用最简单的数据增强方式 `RandomResizedCrop`
- 关闭模型中的任何regularization部分，例如`droppath`
- 关闭`mixup`

单机单卡验证torch和oneflow在上述条件下的收敛情况，先确认是不是模型的问题

### TODO
单机单卡下在imagenette数据集验证
- [x] 用models仓库下的脚本直接跑Swin-tiny模型，查看是否在imagenette上是否可以正常收敛
  - [x] 关闭drop-path的收敛情况（可以正常收敛）
  - [x] 开启drop-path的收敛情况（可以正常收敛）
  - [x] 使用AdamW优化器的收敛情况（可以正常收敛）

单机单卡下在imagenet数据集上验证
- [ ] 改写一下dataloader，在正常的imagenet上训练，只用最简单数据增强，尝试一下是否可以正常收敛
- [ ] 加上数据增强，查看是否可以正常收敛
- [ ] 加上mixup等查看是否可以正常收敛

**Comment by kaijieshi7** [comment]
1. 修复了drop_path传参的bug https://github.com/Oneflow-Inc/models/pull/272

**Comment by kaijieshi7** [comment]
我这边这个模型的torch和oneflow的结果是不一样的，你们看看是不是这样
```
import oneflow as flow
import oneflow.nn as nn
# import torch as flow
# import torch.nn as nn


class N(nn.Module):
    def __init__(self):
        super(N, self).__init__()
        self.proj = nn.Conv2d(3, 96, kernel_size=(4, 4), stride=(4, 4))
        self.norm = nn.LayerNorm(96)
        nn.init.constant_(self.proj.bias, 0.1)
        nn.init.constant_(self.proj.weight, 0.9)
        nn.init.constant_(self.norm.weight, 0.9)
        nn.init.constant_(self.norm.weight, 0.9)

    def forward(self, x):
        x = self.proj(x).flatten(2).transpose(1, 2)
        x = self.norm(x)
        return x

n = N().cuda()
loss_fn = nn.MSELoss()
optim = flow.optim.SGD(n.parameters(), lr=0.1)
optim.zero_grad()
x = flow.ones(2, 3, 224, 224).cuda()
y = n(x)
loss = loss_fn(y, flow.ones(2, 3136, 96).cuda()).backward()
optim.step()
print(n.proj.weight.grad)
print(1)
